### PR TITLE
Creating and Connecting Maria Database to a PHP registration form

### LINKED ISSUE #1018: Creating and connecting Maria Database to a PHP registration form
### Brief Summary: 
The database is a set of clear data stored in a computer. It is used to store information that is very important to an organization. it can hold one or more tables.  In PHP Mysql query_function is used to create a MySQL database. MySQL is the most popular database system used with PHP.

### Key Takeaways: 
After reading this article the reader should be able to:
1. install Xampp
2. create tables in the database using SQL commands
3. store data in a database
4. fetch the stored data from the database and display them on a website
5. create a registration form using bootstrap and CSS

### References: 
N/A

### PR DESCRIPTION
**Pull request by Erastuskanyatta**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
This fixes #1018

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by Erastuskanyatta** [comment]
@hectorkambow. Any Progress?

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by WanjaMIKE** [comment]
@Erastuskanyatta 
In progress.

**Comment by Erastuskanyatta** [comment]
@WanjaMIKE  
Thankyou

File: articles/mariadb/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-### Creating, Connecting, Storing, and fetching data from MariaDatabase using PHP 
+## Creating, Connecting, Storing, and fetching data from MariaDatabase using PHP 
 
 Working with the Database seems to be very complicated in a Developer's coding life. But to me, this is not the case. In this article, we will prove how easy it is to work with MariaDatabase with only SQL and PHP knowledge.
```
Comment (WanjaMIKE) [inline articles/mariadb/index.md:3]: In the Introduction, could you please discuss more what MariaDB entails

File: articles/mariadb/index.md [hunk 2]
```diff
@@ -26,17 +26,17 @@ In this article you will learn how to;
 
 - fetch the saved data from the Database  
 
-### Step 1: Installing Xampp
+## Step 1: Installing Xampp
```
Comment (WanjaMIKE) [inline articles/mariadb/index.md:29]: The preferred heading is `###` rather than `##`

File: articles/mariadb/index.md [hunk 3]
```diff
@@ -26,17 +26,17 @@ In this article you will learn how to;
...
 
 Xampp comes along with PhpMyAdmin when downloaded into a machine.  PhpMyAdmin is the MariaDatabase that we will be working on within this article.
```
Comment (WanjaMIKE) [inline articles/mariadb/index.md:31]: Regarding this sentence
`PhpMyAdmin is the MariaDatabase that we will be working on within this article.`
PhpMyAdmin is merely the tool to help manage the database. Make correction

File: articles/mariadb/index.md [hunk 4]
```diff
@@ -26,17 +26,17 @@ In this article you will learn how to;
...
 
 To install Xampp you need to;
 
-### . Download the package to be installed.
+## . Download the package to be installed.
```
Comment (WanjaMIKE) [inline articles/mariadb/index.md:35]: No need for this section instead go straight to the contents. Remove the heading.

File: articles/mariadb/index.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
-### Creating, Connecting, Storing, and fetching data from MariaDatabase using PHP 
+## Creating, Connecting, Storing, and fetching data from MariaDatabase using PHP 
 
 Working with the Database seems to be very complicated in a Developer's coding life. But to me, this is not the case. In this article, we will prove how easy it is to work with MariaDatabase with only SQL and PHP knowledge.
 
-### Prerequisites
+## Prerequisites
 
 To use this tutorial you need to ;
```
Comment (WanjaMIKE) [inline articles/mariadb/index.md:8]: Begin all the sentences in the list with a capital letter.

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]
Nice article @Erastuskanyatta. Quite Informative and easy to understand. Cold you please the following minor changes. Also, inform the readers in the intro that your are using linux. Thanks

**Comment by Erastuskanyatta** [comment]
@WanjaMIKE  minor changes completed. Please proceed.

**Review by WanjaMIKE — APPROVED** [review]

**Pull request merged**